<ui version="4.0" >
 <class>QStyleEditor</class>
 <widget class="QDialog" name="QStyleEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Help</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTextEdit" name="textEdit" />
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="applyButton" >
         <property name="text" >
          <string>&amp;Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoApply" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="text" >
          <string>Auto Apply</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton" >
         <property name="text" >
          <string>&amp;Load File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
